### PR TITLE
Fix "required attributes are not set [retryCount]" error

### PR DESCRIPTION
**Pull request by frsyuki**
This happens if REST API tried to get tasks of an very old attempt that
did not have retryCount field. This change solves it by setting a
default value.

### CODE REVIEW

File: digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java [hunk 1]
```diff
@@ -149,7 +151,9 @@ public DatabaseSessionStoreManager(DBI dbi, ConfigFactory cf, ConfigMapper cfm,
         dbi.registerMapper(new InstantMapper());
         dbi.registerArgumentFactory(cfm.getArgumentFactory());
 
-        this.mapper = mapper;
+        this.taskArchiveMapper = mapper.copy();
+        this.taskArchiveMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```
Comment (komamitsu) [inline digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java]: 👍

**Comment by komamitsu** [comment]
👍

File: digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java (hunk 1 above)
Comment (frsyuki) [inline digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java]: @komamitsu In fact, this line is not the main fix. This line is for this: https://github.com/treasure-data/digdag/pull/467/files#diff-b8b63c99b1a34f8b65c161e6885fa07cR477
The main fix (fix for adding columns) is this: https://github.com/treasure-data/digdag/pull/467/files#diff-a6a683385d89bdbb7743550326a8645eR38

**Comment by frsyuki** [comment]
👍

**Pull request merged**